theme/assets/js/templates/store/item.js

```diff
@@ -495,6 +497,103 @@ document.addEventListener('alpine:init', () => {
         },
     });
 
+    const createAfterpayBadgeData = () => ({
+        payments: null,
+        paymentAmount: null,
+        isLoadingAfterpayBadge: false,
+        selector: '.afterpay-placement',
+        /**
+         * Initialize Square Web Payments SDK
+         *
+         * @param {String} applicationId the application id
+         * @param {String} locationId the location id
+         * @return {Object|null}
+        */
+        initWebPaymentsSDK(applicationId, locationId) {
+            if (!this.webPaymentsSDKLoaded()) {
+                throw new Error('Square Web Payments SDK is not loaded');
+            }
+
+            const Square = window.Square;
+            return Square.payments(applicationId, locationId);
+        },
+
+        /**
+         * Whether the Web Payments SDK has already been loaded
+         * @return {Boolean}
+         */
+        webPaymentsSDKLoaded() {
+            return !!window.Square;
+        },
+        async init() {
+            // Initialize the payments SDK
+            const applicationId = Alpine.store('global').applicationId;
```

Comment:
Can we use const since it's used multiple time?
```suggestion
            const globalStore = Alpine.store('global');
            const applicationId = globalStore.applicationId;
```